app/src/main/AndroidManifest.xml

```diff
@@ -63,6 +69,7 @@
             </intent-filter>
         </receiver>
 
+        <receiver android:name=".receiver.ActionsReceiver"/>
```

Comment:
The intent filter should be here in the manifest.  The way you did this the receiver can only be invoked when the MainActivity opens and in this state it's also effectively leaking the receiver.